transformations/emoji_icon_transformation/text2icon.json

```diff
@@ -0,0 +1 @@
+{"smiley": [":\u2011)", ":)", ":-]", ":]", ":-3", ":3", ":->", ":>", "8-)", "8)", ":-}", ":}", ":o)", ":c)", ":^)", "=]", "=)"], "laughing": [":\u2011D", ":D", "8\u2011D", "8D", "x\u2011D", "xD", "X\u2011D", "XD", "=D", "=3", "B^D", "c:", "C:"], "sad": [":\u2011(", ":(", ":\u2011c", ":c", ":\u2011<", ":<", ":\u2011[", ":[", ":(", ";("], "angry": [">:[", ":{", ":@"], "crying": [":'\u2011(", ":'("], "tears_of_happiness": [":'\u2011)", ":')"], "disgust": ["D\u2011':", "D:<", "D:", "D8", "D;", "D=", "DX"], "surprise": [":\u2011O", ":O", ":\u2011o", ":o", ":-0", "8\u20110", ">:O"], "kiss": [":-*", ":*", ":\u00d7"], "wink": [";\u2011)", ";)", "*-)", "*)", ";\u2011]", ";]", ";^)", ";>", ":\u2011,", ";D"], "tongue": [":\u2011P", ":P", "X\u2011P", "XP", "x\u2011p", "xp", ":\u2011p", ":p", ":\u2011\u00de", ":\u00de", ":\u2011\u00fe", ":\u00fe", ":\u2011b", ":b", "d:", "=p", ">:P"], "skeptical": [":-/", ":/", ":\u2011.", ">:\\", ">:/", ":\\", "=/", "=\\", ":L", "=L", ":S"], "straight_face": [":\u2011|", ":|"], "embarassed": [":$", "://)", "://3"], "sealed_lips": [":\u2011X", ":X", ":\u2011#", ":#", ":\u2011&", ":&"], "angel": ["O:\u2011)", "O:)", "0:\u20113", "0:3", "0:\u2011)", "0:)", "0;^)"], "evil": [">:\u2011)", ":)", "}:\u2011)", "}:)", "3:\u2011)", "3:)", ">;)", ">:3", ";3"], "cool": ["|;\u2011)", "B-)"], "bored": ["|\u2011O"], "tongue_in_cheek": [":\u2011J"], "confused": ["%\u2011)", "%)"], "sick": [":\u2011###..", ":###.."], "disbelief": ["',:-|", "',:-l"], "awkward": [":E"], "skull": ["8-X", "8=X", "x-3", "x=3"]}
```

Comment:
Is there support for more emojis than these ones perhaps? there are more emojis than these ones, I suppose?

Reply:
The transformation is written so that new emojis and icons can be easily added just by updating the JSON files. I have used [Wikipedia](https://en.wikipedia.org/wiki/List_of_emoticons) as a reliable source of equivalence between emojis and icons but I agree that many more emojis could be added, if there is at least one icon that is used with the same meaning. I believe that the most used ones are in the JSON files right now but I am available to update them with new ones in the future.